src/table/array_table.cpp

```diff
@@ -19,14 +19,17 @@ ArrayWorker<T>::ArrayWorker(size_t size) : WorkerTable(), size_(size) {
   Log::Debug("worker %d create arrayTable with %d elements.\n", MV_Rank(), size);
 }
 
+template <typename T>
+ArrayWorker<T>::ArrayWorker(const ArrayTableInitOption &option) {
+  new (this)ArrayWorker(option.size);
```

Comment:
it's strange. 

I think the following should be fine. 

ArrayWorker<T>::ArrayWorker(const ArrayTableOption &option) : ArrayWorker(option.size) {
}